velox/substrait/SubstraitToVeloxExpr.h

```diff
@@ -40,12 +40,12 @@ class SubstraitVeloxExprConverter {
       const RowTypePtr& inputType);
 
   /// Convert Substrait ScalarFunction into Velox Expression.
-  std::shared_ptr<const core::ITypedExpr> toVeloxExpr(
+  core::TypedExprPtr toVeloxExpr(
```

Comment:
Thank you for refactor. It would be nice to submit it as a separate PR to keep changes in this PR small.

Reply:
Sure. Submitted https://github.com/facebookincubator/velox/pull/5177.